kogito-infinispan-persistence-springboot/README.md

```diff
@@ -17,15 +17,56 @@ This example shows
 
 It utilizes Infinispan server as the backend store. 
 	
-	
+* Process	
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
+
+* Process Properties	
+<p align="center"><img src="docs/images/processProperties.png"></p>
+
+* Process Properties	
+<p align="center"><img src="docs/images/processProperties2.png"></p>
+
+* Process Properties	
+<p align="center"><img src="docs/images/processProperties3.png"></p>
+
+* Call a deal	
+<p align="center"><img src="docs/images/callADeal.png"></p>
+
+* Call a deal	
+<p align="center"><img src="docs/images/callADeal2.png"></p>
+
+* Print review the Deal	
+<p align="center"><img src="docs/images/printReviewTheDeal.png"></p>
+
+* Subprocess
 <p align="center"><img width=75% height=50% src="docs/images/subprocess.png"></p>
 
+* Deal Review	
+<p align="center"><img src="docs/images/dealReview.png"></p>
+
+* Deal Review	
+<p align="center"><img src="docs/images/dealReview2.png"></p>
+
+* Deal Review	
+<p align="center"><img src="docs/images/dealReview3.png"></p>
+
+* Review deal user task	
+<p align="center"><img src="docs/images/reviewDealUserTask.png"></p>
+
+* Review deal user task	
+<p align="center"><img src="docs/images/reviewDealUserTask2.png"></p>
```

Comment:
Please only show the properties panel, as for other screenshots

Reply:
Done @radtriste